tests/runtime_errors/test_dictionary_01.py

```diff
@@ -0,0 +1,8 @@
+from ltypes import i32
+
+def test_dictionary():
+    x: dict[i32, str]
+    x = {1: "one", 2: "two"}
+    print(x[10])
```

Comment:
For the following condition:
https://github.com/lcompilers/lpython/blob/16873d03cfab100232602292a5ba8c78ae0e9b27/src/libasr/codegen/llvm_utils.cpp#L1495-L1522
is this the correct test?
Or 
How should I test KeyError?
@czgdp1807

Reply:
But it prints:
```console
$ lpython tests/runtime_errors/test_dictionary_01.py
(null)
```
Is there any other tests?